plugins/module_utils/crypto/cryptography_crl.py

```diff
@@ -1,19 +1,7 @@
 # -*- coding: utf-8 -*-
 #
 # (c) 2019, Felix Fontein <felix@fontein.de>
-#
-# Ansible is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# Ansible is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
It would be helpful if you could label files with a comment such as `# SPDX-License-Identifier: GPL-3.0-or-later` or `# SPDX-License-Identifier: BSD-2-Clause`. License scanners can detect the previous license header as `GPL-3.0-or-later`, but not the new one (at least not consistently), as it doesn't follow any standardized format.

I would also recommend looking into the [REUSE.software](https://REUSE.software) standard. KDE and the Linux kernel, amongst others, follow this, at least to some extent.

Reply:
Even if you don't want to add an `SPDX-License-Identifier` comment to every file, like REUSE dictates, you could start by changing the name of the `licenses` directory to `LICENSES` and change `PSF-license.txt` > `PSF-2.0.txt` and `simplified_bsd.txt` > `BSD-2-Clause.txt`.

Reply:
Hmm, this simplified header is standard in ansible-core and a lot of other collections (like community.general). I don't like the idea of replacing it (or the copyright statements above it) completely with the notation in https://reuse.software/spec/, especially since the current simplified version is a lot easier to read for users. But I guess it would be OK to add the `SPDX-License-Identifier` line directly after this simplified header.

Also the `licenses` directory and the filenames in there are as in ansible-core (https://github.com/ansible/ansible/tree/devel/licenses), and the main project license is in `COPYING` outside this directory.

Reply:
I created https://github.com/ansible-community/community-topics/issues/112 to discuss this.